api/v1alpha7/types.go

```diff
@@ -29,6 +29,11 @@ type ExternalRouterIPParam struct {
 	Subnet SubnetFilter `json:"subnet"`
 }
 
+type WorkerFloatingIPConfig struct {
+	Enabled     bool     `json:"enabled,omitempty"`
```

Comment:
Do we even need this flag? Could a `nil` or empty list work as a disabled state?